app-ui-catalog/src/main/java/app/k9mail/ui/catalog/ui/organism/items/AppBarItems.kt

```diff
@@ -1,11 +1,11 @@
 package app.k9mail.ui.catalog.ui.organism.items
 
 import androidx.compose.foundation.lazy.grid.LazyGridScope
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.vector.ImageVector
 import app.k9mail.core.ui.compose.designsystem.atom.button.ButtonIcon
 import app.k9mail.core.ui.compose.designsystem.atom.icon.Icons
+import app.k9mail.core.ui.compose.designsystem.organism.SubtitleTopAppBar
```

Comment:
This file is in src/main/java as well, can we move it over?

Reply:
I think the whole catalog app could be moved to the kotlin source set, there is no Java left.

Reply:
Move the app to kotlin source set